frameworks/cassandra/src/main/dist/svc.yml

```diff
@@ -313,7 +534,35 @@ pods:
                     cqlsh -u cassandra -p cassandra {{{CASSANDRA_CQLSH_SSL_FLAGS}}} -e "alter keyspace system_traces      WITH replication = {'class': 'SimpleStrategy', 'replication_factor': 3};" "node-0-server.${FRAMEWORK_HOST}" {{TASKCFG_ALL_CASSANDRA_NATIVE_TRANSPORT_PORT}} &&
                     cqlsh -u cassandra -p cassandra {{{CASSANDRA_CQLSH_SSL_FLAGS}}} -e "alter keyspace system_auth        WITH replication = {'class': 'SimpleStrategy', 'replication_factor': 3};" "node-0-server.${FRAMEWORK_HOST}" {{TASKCFG_ALL_CASSANDRA_NATIVE_TRANSPORT_PORT}} &&
                     cqlsh -u cassandra -p cassandra {{{CASSANDRA_CQLSH_SSL_FLAGS}}} -e "alter keyspace system_distributed WITH replication = {'class': 'SimpleStrategy', 'replication_factor': 3};" "node-0-server.${FRAMEWORK_HOST}" {{TASKCFG_ALL_CASSANDRA_NATIVE_TRANSPORT_PORT}} &&
-                    nodetool repair system_auth &&
+                    {{#TASKCFG_ALL_SECURE_JMX_ENABLED}}
+                    ./bootstrap --resolve=false
```

Comment:
you're already calling this a [few lines up](https://github.com/mesosphere/dcos-commons/pull/3146/files#diff-fe0c20da1fba698643bc101afa99c204R529)

Also does all of this code get executed? The beginning of this has `cmd: >` and every line has either a `;` or `&&` while this new section does not. I wonder if this code path is getting tested?